src/main/webapp/content/scss/global.scss

```diff
@@ -752,6 +752,7 @@ Course Info Bar
 
     img {
         max-width: 100%;
+        height: auto;
```

Comment:
Good fix to keep the aspect ratio. Also the solution used here: https://html.com/attributes/img-height/